README.md

```diff
@@ -1336,6 +1336,7 @@ Software that does not fit in another section.
 
 _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-foss/awesome-sysadmin#it-asset-management)_
 
+- [Actual Budget](https://actualbudget.com) - This is Actual, a local-first personal finance tool. It is now 100% free and open-source. ([Demo](https://app.actualbudget.com/budget) [Source Code](https://github.com/actualbudget/actual-server)) `MIT` `JavaScript`
```

Comment:
- Please add the missing comma after the demo link

`([Demo](https://app.actualbudget.com/budget), [Source Code](https://github.com/actualbudget/actual-server))`

Reply:
Fixed as per suggested edits, thanks.